client/components/deposits-overview/test/index.tsx

```diff
@@ -214,6 +248,31 @@ describe( 'Deposits Overview information', () => {
 		expect( getByText( 'Estimated' ) ).toBeTruthy();
 		expect( getByText( '—' ) ).toBeTruthy();
 	} );
+
+	test( 'Confirm recent deposits renders ', () => {
+		mockUseDeposits.mockReturnValue( {
+			depositsCount: 0,
+			deposits: mockDeposits,
+			isLoading: false,
+		} );
+		const { getByText } = render(
+			<RecentDeposits currency={ mockAccount.default_currency } />
+		);
+		expect( getByText( 'January 2, 2020' ) ).toBeTruthy();
```

Comment:
Note: [`getByText` will fail the test if it can't find an element](https://testing-library.com/docs/queries/about#types-of-queries) (or there is > 1), so `toBeTruthy` is not required.

```suggestion
		getByText( 'January 2, 2020' );
```